### ISSUE TITLE
Can't update model passing an Array as second param

### ISSUE THREAD

**Issue by balaguero** [balderdashy/sails-mongo#368]
Hi. I'm trying to update a model.
If i run this code (passing an object as Update second parameter)
```javascript
UserProfile
.update({id: 1}, {phone: '555-12345'})
.exec(function(err, updatedProfile){
  console.log(err, updatedProfile);
});
```
It works right!

But if i run this code (passing an Array as Update second parameter)
```javascript
var thingsToUpdate = [
  {name:      updatedInfo.name},
  {lastName:  updatedInfo.lastName},
  {phone:     updatedInfo.phone} 
];
UserProfile
.update({id: 1}, thingsToUpdate )
.exec(function(err, updatedProfile){
  console.log(err, updatedProfile);
});
```
It doesn't works and i get this error message:

>Error (E_UNKNOWN) :: Encountered an unexpected error
MongoError: '$set' is empty. You must specify a field like so: {$set: {<field>: ...}}
    at Function.MongoError.create (D:\node-projects\bitsprofile\node_modules\sails-mongo\node_modules\mongodb\node_modules\mongodb-core\lib\error.js:31:11)
    at toError (D:\node-projects\bitsprofile\node_modules\sails-mongo\node_modules\mongodb\lib\utils.js:114:22)
    at D:\node-projects\bitsprofile\node_modules\sails-mongo\node_modules\mongodb\lib\collection.js:972:67
    at D:\node-projects\bitsprofile\node_modules\sails-mongo\node_modules\mongodb\node_modules\mongodb-core\lib\topologies\server.js:778:13
    at Callbacks.emit (D:\node-projects\bitsprofile\node_modules\sails-mongo\node_modules\mongodb\node_modules\mongodb-core\lib\topologies\server.js:95:3)
    at null.messageHandler (D:\node-projects\bitsprofile\node_modules\sails-mongo\node_modules\mongodb\node_modules\mongodb-core\lib\topologies\server.js:249:23)
    at Socket.<anonymous> (D:\node-projects\bitsprofile\node_modules\sails-mongo\node_modules\mongodb\node_modules\mongodb-core\lib\connection\connection.js:262:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at TCP.onread (net.js:523:20)
Details:  MongoError: '$set' is empty. You must specify a field like so: {$set: {<field>: ...}}

**Comment by devinivy** [comment]
Try this instead,
```js
var thingsToUpdate = {
  name:      updatedInfo.name,
  lastName:  updatedInfo.lastName,
  phone:     updatedInfo.phone 
};
```

If there was any faulty or misleading documentation, definitely let us know!

**Comment by sailsbot** [comment]
Thanks for posting, @balaguero.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
